app/experimenter/experiments/tests/api/v5/test_serializers.py

```diff
@@ -1069,7 +1069,13 @@ def test_name_change_with_preview_status(self):
             serializer.errors,
         )
 
-    def test_status_to_review_generates_bucket_allocation(self):
+    @parameterized.expand(
+        [
+            [NimbusExperiment.Status.REVIEW],
+            [NimbusExperiment.Status.PREVIEW],
+        ]
+    )
+    def test_status_generates_bucket_allocation(self, to_status):
```

Comment:
Man parameterized tests are so nice.  Shout out to @wolever for making our lives so much better ❤️